dags/synapse-dataset-to-croissant.py

```diff
@@ -352,15 +327,14 @@ def construct_distribution_section_for_files(files_attached_to_dataset: List[Fil
     id_and_version_pairs = []
     for file in files_attached_to_dataset:
         file: File = file
-        if not file.file_handle or not file.file_handle.content_md5:
-            modified_file_id = int(file.id.replace("syn", ""))
-            ids_of_files.append(modified_file_id)
-            id_and_version_pairs.append(modified_file_id)
-            id_and_version_pairs.append(file.version_number)
-            files_to_find_md5_in_snowflake[int(file.id.replace(
-                "syn", ""))] = file.version_number
-
-    file_md5s = None
+        modified_file_id = int(file.id.replace("syn", ""))
+        ids_of_files.append(modified_file_id)
+        id_and_version_pairs.append(modified_file_id)
+        id_and_version_pairs.append(file.version_number)
+        files_to_find_md5_in_snowflake[int(file.id.replace(
+            "syn", ""))] = file.version_number
```

Comment:
Moving to just pull this information from snowflake. Background comment: https://github.com/Sage-Bionetworks-Workflows/orca-recipes/pull/83#discussion_r2031799819